packages/mdc-checkbox/index.js

```diff
@@ -93,39 +93,59 @@ class MDCCheckbox extends MDCComponent {
       setNativeControlAttr: (attr, value) => this.nativeCb_.setAttribute(attr, value),
       removeNativeControlAttr: (attr) => this.nativeCb_.removeAttribute(attr),
       getNativeControl: () => this.nativeCb_,
+      isIndeterminate: () => this.isIndeterminate_(),
+      isChecked: () => this.isChecked_(),
+      hasNativeControl: () => !!this.nativeCb_,
+      setNativeControlDisabled: (disabled) => this.nativeCb_.disabled = disabled,
       forceLayout: () => this.root_.offsetWidth,
       isAttachedToDOM: () => Boolean(this.root_.parentNode),
     });
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  isIndeterminate_() {
+    return this.nativeCb_.indeterminate;
```

Comment:
Can we move these to the `get` methods and then update the adapter to use the public `get` ? Seems like we don't really need the private function.